host_core/lib/host_core.ex

```diff
@@ -129,12 +129,30 @@ defmodule HostCore do
         end
       end
 
+    host_config =
+      if config.host_config == nil do
```

Comment:
nit: it's a little more idiomatic to use `is_nil/1` here

Reply:
Thanks! I used `is_nil/2` now